### ISSUE TITLE
Wrong Stats and Headings

### ISSUE THREAD

**Issue by cayars** [ecleese/plexWatchWeb#104]
I'm noticing some strange things.  Here is the text version of my stats:
8 2D/3D MVC Movies <-- correct
140 3D Movies <-- correct
27 Educational <-- showing number of shows not episodes which should be 1132
1132 TV Episodes <--This is actually the Education episode count for above
4761 Movies <-- correct
3174 Music <--correct number of artists but shouldn't it show albums instead?
37 NFL <--showing number of "shows" and not espisodes
837 TV Episodes <--this is the NFL episode count for above
10 Screeners <-- correct
118 TV Shows <-- correct
8795 TV Episodes <-- correct

So basically it's a naming issue for both the Educational and NFL sections.  These are setup in plex as TV Shows but use a customer scanner.  Numbers are correct but the stats page should use the name correctly instead of what appears to be hard coded "TV Episodes".

**Comment by Arcanemagus** [comment]
How would you propose to fix this? The default library name for TV is "TV Shows" so you can't simply go $section['title'] . "Shows" as that would give you "TV Shows Shows" (Although it would show "Educational Shows").

The same goes for the number of episodes, the default library screws everything up again since we can't simply append "Episodes" on the end of the section as you would get "TV Shows Episodes" which isn't as bad as the section label, but isn't ideal.

As for the Music one could argue that the number of tracks would be better, I'm sure there are proponents for Artists / Albums / Tracks.

----

I'm currently thinking that we match on the $section['title'] and if it matches "TV Shows" then we leave it alone, otherwise we append "Shows" for the whole section. For the episode count we do something similar to hardcode "TV Episodes" and otherwise append "Episodes".

The issue with this is whether Plex ever localizes that, and when people do their own name.

An alternative is a different layout, maybe something like this:

    Shows     |      27     |     37      |     118     |             |             |             |             |             |
    ----------|-------------|-------------|-------------|      8      |      10     |     140     |     4761    |     3174    |
    Episodes  |     1132    |     837     |    8795     |             |             |             |             |             |
    ---------------------------------------------------------------------------------------------------------------------------
              | Educational |     NFL     |   TV Shows  |  2D/3D MVC  |  Screeners  |  3D Movies  |    Movies   |     Music   |
              |             |             |             |    Movies   |             |             |             |             |

**Comment by Arcanemagus** [comment]
Created a pull request that does the simple name fixing I proposed above. This would (for example) make your list of TV libraries show up as:

* 27 Educational Shows
* 1132 Educational Episodes
* 37 NFL Shows
* 837 NFL Episodes
* 118 TV Shows
* 8795 TV Episodes

Due to the way Plex structures it's data getting anything other than the number of artists from the Music library would be rather involved as it only gives the total artists at the library level.

I'll experiment a bit with alternative layouts to see if any could show this data in a better manner. If you would like to test this out for yourself just replace the datafactory/get-library-stats.php file in your plexWatch installation with https://github.com/Arcanemagus/plexWatchWeb/raw/header-stat-names/datafactory/get-library-stats.php.

**Comment by Arcanemagus** [comment]
@cayars can you try out the current version (v1.7.0)? This has the changes outlined above that simply fix the name. If you have further suggestions as to how that should be changed please elaborate :wink:

**Comment by Arcanemagus** [comment]
Since we haven't gotten any feedback and this is fixed in a basic manner I'm closing this issue. Feel free to add more details and re-open if you feel the need.
